ml-agents/mlagents/trainers/sac/optimizer_torch.py

```diff
@@ -202,10 +202,22 @@ def sac_q_loss(
         return q1_loss, q2_loss
 
     def soft_update(self, source: nn.Module, target: nn.Module, tau: float) -> None:
```

Comment:
I would make this method static (and even move it to ModelUtils)

Reply:
Moved, and added test